tests/pos/t8230a.scala

```diff
@@ -15,12 +15,12 @@ object Test {
   object Okay {
     Arr("1")
 
-    import I.{ arrToTrav, longArrToTrav }
-    foo(Arr("2"))
-  }
+   import I.{ arrToTrav, longArrToTrav }
+   val x = foo(Arr("2"))
+ }
 
   object Fail {
-    import I.arrToTrav
-    foo(Arr("3")) // found String, expected Long
+//    import I.arrToTrav
+//    foo(Arr("3")) // found String, expected Long
```

Comment:
Does this require further fixes before it compiles? If so, leave a breadcrumb by splitting this part out into a file in `pending/pos`.